lib/archethic/metrics/metric_client.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArchEthic.Metrics.MetricClient do
+  @moduledoc """
+  gensever
+  Visit dif
+  """
+  use GenServer
+  @process_name __MODULE__
+  @default_state %{
+    counter: 0
+  }
+  def start_link(_init_arg) do
+    GenServer.start_link(__MODULE__, @default_state, name: @process_name)
+  end
+
+  def init(_state) do
+    {:ok, @default_state}
+  end
+
+  def get_this_node_points() do
+    ArchEthic.Metrics.MetricNodePoller.get_points()
+  end
+
+  def get_network_points() do
+    ArchEthic.Metrics.MetricNetworkPoller.get_points()
+  end
+
+  def monitor() do
```

Comment:
maybe rename it to `subscribe_to_updates`

Reply:
noted